### PR TITLE
Replaces the otc pair field with two asset components

### LINKED ISSUE #2240: Change otc form to use asset components instead of pair
## Abstract
Manually inputting asset pairs is cumbersome

### Task
- [ ] Replace the current Pair in the UI with two asset selectors

### PR DESCRIPTION
**Pull request by kelsos**
Closes #2240

### CODE REVIEW

File: docs/changelog.rst [hunk 1]
```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :feature:`2240` Users now can select the supported assets from a dropdown when creating and otc trade.
```
Comment (LefterisJP) [inline docs/changelog.rst:5]: It's now call external trade. So `and otc` -> `an external`

**Review by LefterisJP — COMMENTED** [review]
Looks good. Seems to work fine.

A nice and small improvement is if you could help the user understand what a buy and a sell means here in regards to the pair. And we should probably also start showing those elsewhere too. In trade tables and other places.

For a buy (please note the image should have FOR -> WITH)



So just: "Buy BASE with QUOTE"

For a sell:



So just: "Sell BASE for QUOTE"

File: docs/changelog.rst (hunk 1 above)
Comment (kelsos) [reply docs/changelog.rst:5]: I also need to rename the components

**Comment by kelsos** [comment]
@LefterisJP an approval plx?

**Comment by LefterisJP** [comment]
Please be pinging me in discord for reviews

**Review by LefterisJP — APPROVED** [review]
LGTM! Can you also perhaps add a hint in the rate? To explain what it is if buy is selected and also what it is if sell is selected?

So for example:

1. Buy ETH with BTC for 0.05 BTC per ETH
2. Sell 10 ETH for EUR for 1000 EUR per ETH.

Not sure where this could be put to look nice to the user but I think it would be a tremendous value add.

Naturally this can happen in another PR.

**Pull request merged**